security/access_token.rst

```diff
@@ -697,6 +697,187 @@ create your own User from the claims, you must
         }
     }
 
+Using CAS 2.0
+-------------
+
+`Central Authentication Service (CAS)`_ is an enterprise multilingual single
+sign-on solution and identity provider for the web and attempts to be a
+comprehensive platform for your authentication and authorization needs.
```

Comment:
I never used CAS, we can probably have a better description

@nacorp your suggestions are welcomed

Reply:
Ow you've been so quick.  #19490  was on top of my todo. I'll come by to check what you wrote and complete if necessary as soon as I can. Thanks a lot for your help :-)

Reply:
@nacorp friendly ping about this in case you have some time to review it. Thanks a lot!

Reply:
I'm on it. The description provided by alamirault is ok. I want to test the documentation he wrote on a real project but I miss some time at the moment.

Reply:
I'm so sorry for being that late - it sound great for me ! (thanks @alamirault  !)